addon/adapters/-addon-docs.js

```diff
@@ -1,5 +1,5 @@
 import DS from 'ember-data';
-import config from 'dummy/config/environment';
+import config from 'ember-get-config';
```

Comment:
This and the many other edits like it are because we don't want to hard-code the name "dummy" as our app's modulePrefix.